pkgs/by-name/da/dashy-ui/package.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  yarnConfigHook,
+  yarnBuildHook,
+  nodejs,
+  yq-go,
+  settings ? { },
+}:
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
I named the package `dashy-ui` rather than `dashy`, similar to `netbird-ui`. I chose `-ui` because it's not exactly the same as `dashy` as it's used in docker; this is a static web app rather than a full node server/package/etc.